src/main/scala/beam/agentsim/agents/modalbehaviors/ModeChoiceCalculator.scala

```diff
@@ -147,9 +152,9 @@ object ModeChoiceCalculator {
         _ =>
           new ModeChoiceUniformRandom(beamServices.beamConfig)
       case "ModeChoiceMultinomialLogit" =>
-        val logit = ModeChoiceMultinomialLogit.buildModelFromConfig(configHolder)
+        val (logit, modeLogit) = ModeChoiceMultinomialLogit.buildModelFromConfig(configHolder)
```

Comment:
might be more informative to name the second one `routeLogit` if it's choosing between several alternative transit trips

Reply:
Actually the first one chooses between several alternative transit trips. So I named the first one routeLogit. Let me know if something is unclear here.